txsni/test/certs/cert_builder.py

```diff
@@ -28,6 +28,13 @@
 HTTP2BIN_CERT_PATH = os.path.join(CERT_DIR, 'http2bin.org.pem')
 HTTP2BIN_KEY_PATH = os.path.join(CERT_DIR, 'http2bin.org.key')
 
+# acme style (not concatenated)
```

Comment:
It might be nice to have these put together in an object that actually represents the "acme style" in one place that isn't in the tests